modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -124,22 +124,51 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 		reverseRelayer sdk.AccAddress
 		forwardRelayer string
 		refundAcc      sdk.AccAddress
+		refundAccBal   sdk.Coin
+		fee            types.Fee
+		packetID       channeltypes.PacketId
 	)
 
 	validSeq := uint64(1)
 
 	testCases := []struct {
-		name     string
-		malleate func()
-		expPass  bool
+		name      string
+		malleate  func()
+		expResult func()
```

Comment:
I made these changes to test `distributeFee`. No longer necessary for this pr, but will be for future pr's (implementing the locking functionality) so I will leave the changes